#! /usr/bin/env python

PACKAGE = "lidar_camera_offline_calibration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pointcloud_input", str_t, 0, "point cloud topic", "/rslidar_points")

gen.add("image_input", str_t, 0, "image topic", "/pylon/raw_image_2")

gen.add("camera_matrix", str_t, 0, "camera matrix 3x3", "475.0, 0.0, 240.0, 0.0, 475.0, 151.0, 0.0, 0.0, 1.0")

exit(gen.generate(PACKAGE, "lidar_camera_offline_calibration", "Calibration"))
